ga4gh/backend.py

```diff
@@ -9,6 +9,7 @@
 import os
 
 import ga4gh.protocol as protocol
+from ga4gh.frontend import app
```

Comment:
We're breaking the isolation between the frontend and backend here. The backend should definitely not depend on the frontend for all sorts of reasons.